Workflow/info.plist

```diff
@@ -1289,6 +1289,27 @@ Query DALL·E via the `dalle` keyword.
 			<key>variable</key>
 			<string>openai_api_key</string>
 		</dict>
+		<dict>
+			<key>config</key>
+			<dict>
+				<key>default</key>
+				<string></string>
+				<key>placeholder</key>
+				<string></string>
+				<key>required</key>
+				<false/>
+				<key>trim</key>
+				<true/>
+			</dict>
+			<key>description</key>
+			<string>Get it at https://platform.openai.com/api-keys</string>
```

Comment:
The URL is incorrect, as this is the one for API keys. Also, this option won’t be relevant to most people so it should be further down.